src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -4877,6 +4877,10 @@ private unsafe char[] GetCanonicalPath(char[] dest, ref int pos, UriFormat forma
                 }
 
                 char[] dest1 = new char[dest.Length];
+                //
```

Comment:
we don't use comment style with empty "//" blocks.

So, this would be a more appropriate style:

```c#
// Some comments to explain things. We try to always use punctuation like periods at the end.
CallImportantMethod();

// More comments to explain things.
CallAnotherMethod();
```

Reply:
Just pushed a commit to fix this.